ResearchKit/Common/ORKDateTimePicker.m

```diff
@@ -35,7 +35,7 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 @interface ORKDateTimePicker ()
 
 @property (nonatomic, strong) ORKAnswerFormat *answerFormat;
-@property (nonatomic, strong) NSCalendar *calendar;
+@property (nonatomic, copy) NSCalendar *calendar;
```

Comment:
Even though NSCalendar implements NSCopying, I don't think it's necessarily a great idea to copy one. For instance an autoupdating calendar passed in might be exactly what's intended.